Neos.Neos/Tests/Behavior/Features/Fusion/FlowQuery.feature

```diff
@@ -318,13 +349,44 @@ Feature: Tests for the "Neos.ContentRepository" Flow Query methods.
     When I execute the following Fusion code:
     """fusion
     test = Neos.Fusion:DataStructure {
+      # @todo Decide wether the changed order of the results compared to Neos 8.3 is ok
+      # Result in Neos 8.3: "typeFilter: a1a,a1b1a,a1a2,a1b2,a1a3,a1a4,a1a5,a1a6"
+      # Result in Neos 9.0: "typeFilter: a1a,a1a2,a1b2,a1a3,a1a4,a1a5,a1a6,a1b1a"
       typeFilter = ${q(node).find('[instanceof Neos.Neos:Test.DocumentType2]').get()}
-      combinedFilter = ${q(node).find('[instanceof Neos.Neos:Test.DocumentType2][uriPathSegment*="b1"]').get()}
+      # @todo Fix and re enable `combinedFilter` case
+      # Result in Neos 8.3: "combinedFilter: a1b1a"
+      # Result in Neos 9.0: "combinedFilter: a1a,a1a2,a1b2,a1a3,a1a4,a1a5,a1a6,a1b1a"
+      # combinedFilter = ${q(node).find('[instanceof Neos.Neos:Test.DocumentType2][uriPathSegment*="b1"]').get()}
+      @process.render = Neos.Neos:Test.RenderNodesDataStructure
+    }
```

Comment:
Should we resolve this todo here or in a followup?

Reply:
Children und find will have to wait for beta 2

Reply:
Might also be that `filter` could benefit from this https://github.com/neos/neos-development-collection/pull/4701

Reply:
okay i thought so already so lets not block this any further ;)